css/style.css

```
@@ -0,0 +1,903 @@
+    font-size: 45px;
+    line-height: 59px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    text-shadow: 4px 8px 10px rgb(0 0 0 / 23%);
+    color: #505050;
+}
+
+.header_text__style span {
+    background-color: #76f4ea;
+    padding: 0 15px;
+}
+
+.header_button {
+    display: flex;
+    justify-content: center;
+    /* padding-top: 12%; */
+    margin-left: 0%;
+}
+
+.header_button__style {
+    display: block;
+    margin: 0 auto;
+    width: 232px;
+    height: 53px;
+    position: relative;
+    font-size: 18px;
+    line-height: 50px;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    color: #000;
+    border: 2px solid #000;
+    border-radius: 2px;
+    text-align: center;
+    transition: .3s;
+}
+
+.header_button__style:hover {
+    cursor: pointer;
+    background-color: rgb(103, 189, 192);
+    transition: .3s;
+    transform: scale(1.1);
+}
+
+.header_button__text {
+    font-family: Montserrat;
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.12em;
+    color: #000000;
+    scroll-behavior: smooth;
+}
+
+.header_image {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 5%;
+    margin-top: -3.1%;
+}
+
+.main-content {
+    padding-top: 10.7%;
+    min-height: 985px;
+    background-color: #f8fafa;
+}
+
+.main-intro {
+    text-align: center;
+    margin-left: 23px;
+}
+
+.main-information {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 141px;
+}
+
+.main-intro__text-par {
+    opacity: 0.9;
+    font-family: Montserrat;
+    font-size: 38px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.043em;
+    color: #000000;
+}
+
+.main-intro__text-content {
+    opacity: 0.61;
+    font-family: Merriweather;
+    font-size: 18px;
+    line-height: 28px;
+    font-weight: 400;
+    text-transform: uppercase;
+    letter-spacing: 0.0699em;
+    color: #000000;
+    margin-top: 11px;
+    margin-left: -21px;
+}
+
+.first-par,
+.second-par {
+    opacity: 0.74;
+    font-family: Merriweather;
+    font-size: 20px;
+    line-height: 28px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 0.068em;
+    color: #000000;
+}
+
+.first-par_text {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.07em;
+    color: #000000;
+    margin-top: 17px;
+}
+
+.second-par_text {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.036em;
+    color: #000000;
+    margin-top: 17px;
+}
+
+.first-par__text-second {
+    opacity: 0.74;
+    font-family: Raleway;
+    font-size: 16px;
+    line-height: 28px;
+    font-weight: 400;
+    letter-spacing: 0.036em;
+    color: #000000;
+    margin-top: 0px;
+}
+
+.second-par {
+    letter-spacing: -0.036em;

```

use other values, this is too small

---

Thanks!
i fixed it!
```suggestion
letter-spacing: 1px;
```

---

Why du you actually need it at all ? Is it very important difference on the screen without it ? :)

---

> Why du you actually need it at all ? Is it very important difference on the screen without it ? :)

No) but using only for pixel perfect)